zeroex/order.go

```diff
@@ -513,10 +513,10 @@ func SignTestOrder(order *Order) (*SignedOrder, error) {
 	return signedOrder, nil
 }
 
-// Trim converts the order to a TrimmedOrder, which is the format expected by
+// Trim converts the order to a LibOrderOrder, which is the format expected by
 // our smart contracts. It removes the ChainID and ExchangeAddress fields.
-func (s *SignedOrder) Trim() wrappers.TrimmedOrder {
-	return wrappers.TrimmedOrder{
+func (s *SignedOrder) Trim() wrappers.LibOrderOrder {
```

Comment:
`abigen` now generates good names for all of the generated types, so it wasn't necessary to alias these names to `TrimmedOrder` anymore.